### ISSUE TITLE
Feature: support cron job 

### ISSUE THREAD

**Issue by Mossaka** [spinkube/containerd-shim-spin#129]
For more info: https://github.com/fermyon/spin-trigger-cron

In general, I think we should reach feature parity between the spin CLI and the spin shim. A table that documents what features are implemented / not implemented is going to be useful

**Comment by tpmccallum** [comment]
**Spin Cron Related:** Does this solve the Cron feature request @Mossaka ?
https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

**Feature Partity Related:** That is an awesome idea!!!
We, at Fermyon, have tables i.e. language support overview. Would something like this inside the SpinKube Documentation work?
https://developer.fermyon.com/spin/v2/language-support-overview

**Comment by devigned** [comment]
I lean more toward K8s for scheduling cron jobs vs using an always running Spin application to run the job. However, I might be missing some benefit of using Spin's cron functionality rather than K8s. Is there a use case that the Spin cron trigger would make more sense than using K8s constructs?

**Comment by kate-goldenring** [comment]
I don't think we want to use K8s cron for this since running a spin app pod is the equivalent of a `spin up`. Running `spin up` on a cron basis wouldn't produce a result. Rather, we may want to support triggering a running app on a cron basis.
